Bottom: 4f35c07ed2d32b60f79605fb01026e028d30ca29
Top:    0c495860cb38f50ac271500f40b1507fb0119461
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:19 +0000

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
